nix/tests/vm-test/default.nix

```diff
@@ -6,19 +6,47 @@ let
     install-default = {
       install = ''
         NIX_PATH=$(readlink -f nix.tar.xz)
-        RUST_BACKTRACE="full" ./nix-installer install --logger pretty --log-directive nix_installer=trace --nix-package-url "file://$NIX_PATH" --no-confirm
+        RUST_BACKTRACE="full" ./nix-installer install --extra-conf 'sandbox = false' --logger pretty --log-directive nix_installer=trace --nix-package-url "file://$NIX_PATH" --no-confirm
```

Comment:
Is it desired that we disable sandboxing in these tests?

Reply:
Oh, my mistake. Thanks!